server/src/main/java/keywhiz/KeywhizConfig.java

```diff
@@ -100,6 +100,9 @@ public class KeywhizConfig extends Configuration {
   @JsonProperty
   private Long maximumSecretSizeInBytesInclusive;
 
+  @JsonProperty
+  private boolean inferSecretOwnerUponCreation;
```

Comment:
We want to be careful about using a boolean here, since we'll be locked into two different options forever. How about something like:

```
enum NewSecretOwnershipStrategy {
  INFER_FROM_CLIENT,
  NONE
}

```